<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CatLineChart</class>
 <widget class="QWidget" name="CatLineChart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="RootLayout">
   <item>
    <widget class="QCatGrayQcustomplotCustomChart" name="ChartWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="FunctionWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QCheckBox" name="Line_0">
          <property name="text">
           <string>Line_0</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="Line_2">
          <property name="text">
           <string>Line_2</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="Line_1">
          <property name="text">
           <string>Line_1</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="Line_3">
          <property name="text">
           <string>Line_3</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="horizontalSpacing">
         <number>10</number>
        </property>
        <property name="verticalSpacing">
         <number>10</number>
        </property>
        <item row="1" column="1">
         <widget class="QComboBox" name="ScatterShapeBox"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="ScatterShapeLabel">
          <property name="text">
           <string>ScatterShape:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="LineStyleLabel">
          <property name="text">
           <string>LineStyle:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="LineStyleBox"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="TracerStyleLabel">
          <property name="text">
           <string>TracerStyle:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="TracerStyleBox"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="PositionTypeLabel">
          <property name="text">
           <string>PositionType:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="PositionTypeBox"/>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>363</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="SaveButton">
          <property name="text">
           <string>SaveImage</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="StartButton">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCatGrayQcustomplotCustomChart</class>
   <extends>QWidget</extends>
   <header location="global">QCatGrayQcustomplotCustomChart.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
